### DISCUSSION TITLE
Custom footer not working

### DISCUSSION THREAD

**Discussion by giuseb** [activeadmin/activeadmin#8586] [Help] [answered]
[According to the docs](https://activeadmin.info/1-general-configuration.html), it should be possible to customize the footer.

However, adding the following to the `active_admin.rb` initializer:

``` ruby
ActiveAdmin.setup do |config|
  config.footer = "My foot!"
end
```

causes the app to crash with:

```
NoMethodError: undefined method `footer=' for an instance of ActiveAdmin::Application (NoMethodError)
```

Should this be reported as a bug, or is the documentation out of sync with the ongoing v4 development?

**Comment by tagliala** [comment, marked as answer]
Hello,

> Should this be reported as a bug, or is the documentation out of sync with the ongoing v4 development?

The documentation is out of sync, but for the moment it should also covers 3.x, so for the moment here it is the correct approach for v4:

Ref: https://github.com/activeadmin/activeadmin/blob/master/UPGRADING.md#from-v3-to-v4-beta

> Now, run `rails g active_admin:views` which will copy the partials to your app so you can customize if needed.

**Comment by giuseb** [comment]
Grazie Geremia!
From the perspective of a new ActiveAdmin user, things are rather confusing at this stage. You are instructed to refer to the documentation, which bears the `4.0.0.beta15` tag, but many topics are, in fact, obsolete.
A notice to that effect would be very useful.
Something like "While the documentation gets updated, please read UPGRADING.md even if you are getting started with activeadmin for the first time".
Would a PR along these lines be welcome?

**Comment by tagliala** [comment]
Ciao Giuseppe,

I am not sure about the exact plans for the documentation. I also have a pending PR (#8543), but since it currently covers version 3.x as well, I am uncertain about its status 

Let's wait for some other feedback
